tests/test_quotequail.py

```diff
@@ -1,1177 +0,0 @@
-                "date": "2016-03-25, at 23:01",
-                "html": '<html><head></head><body style="word-wrap: break-word; -webkit-nbsp-mode: space; -webkit-line-break: after-white-space;" class=""><div class=""><div><div><div class=""><div style="word-wrap: break-word; -webkit-nbsp-mode: space; -webkit-line-break: after-white-space;" class="">Some <b class="">important</b> email</div></div></div></div></div></body></html>',
-                "html_top": '<html><head></head><body style="word-wrap: break-word; -webkit-nbsp-mode: space; -webkit-line-break: after-white-space;" class="">Foo<div class=""><br class=""></div><div class="">Bar</div></body></html>',
-            },
-        )
-
-    def test_gmail_reply(self):
-        html = """<html><head></head><body><div dir="ltr">foo<div><br></div><div>bar</div></div><div class="gmail_extra"><br><div class="gmail_quote">On Wed, Mar 16, 2016 at 12:49 AM, Foo Bar <span dir="ltr">&lt;<a href="mailto:foo@bar.example" target="_blank">foo@bar.example</a>&gt;</span> wrote:<br><blockquote class="gmail_quote" style="margin:0 0 0 .8ex;border-left:1px #ccc solid;padding-left:1ex">Hi,<br>
-<br>This is the reply<br>
-<br>
-Thanks a lot!<br>
-<span class="HOEnZb"><font color="#888888">Foo<br>
-<br>
-</font></span></blockquote></div><br><br clear="all"><div><br></div>-- <br><div class="gmail_signature"><div dir="ltr"><div><div dir="ltr"><b>John Doe</b></div><div dir="ltr"><b>Senior Director</b><div>Some Company</div></div></div></div></div>
-</div>
-</body></html>"""
-
-        self.assertEqual(
-            unwrap_html(html),
-            {
-                "type": "reply",
-                "from": "Foo Bar <foo@bar.example>",
-                "date": "Wed, Mar 16, 2016 at 12:49 AM",
-                "html_top": '<html><head></head><body><div dir="ltr">foo<div><br></div><div>bar</div></div></body></html>',
-                "html": '<html><head></head><body><div class="gmail_extra"><div class="gmail_quote"><div>Hi,<br>\n<br>This is the reply<br>\n<br>\nThanks a lot!<br>\n<span class="HOEnZb"><font color="#888888">Foo</font></span></div></div></div></body></html>',
-                "html_bottom": '<html><head></head><body><div class="gmail_extra">-- <br><div class="gmail_signature"><div dir="ltr"><div><div dir="ltr"><b>John Doe</b></div><div dir="ltr"><b>Senior Director</b><div>Some Company</div></div></div></div></div>\n</div>\n</body></html>',
-            },
-        )
-
-    def test_outlook_forward(self):
-        data = self.read_file("outlook_forward.html")
-        result = unwrap_html(data)
-        self.assertEqual(result["type"], "forward")
-        self.assertEqual(result["from"], "John Doe")
-        self.assertEqual(result["to"], "Foo Bar (foo@bar.example)")
-        self.assertEqual(result["date"], "Wednesday, July 09, 2014 10:27 AM")
-        self.assertEqual(result["subject"], "The subject!")
-        self.assert_equal_to_file(
-            result["html"], "outlook_forward_unwrapped.html"
```

Comment:
are these files still used somewhere or were they inlined?

Reply:
In https://github.com/closeio/quotequail/commit/46864a203278c1d7c685e9660d0e8e7bc4efc6ec I added `tests/test_unwrap_html.py` but forgot to remove the tests. All the `unwrap_html` tests I'm removing here should be there.